lib/ansible/modules/windows/win_scheduled_task.ps1

```diff
@@ -77,25 +78,41 @@ $description = Get-AnsibleParam -obj $params -name "description" -type "str" -de
 $enabled = Get-AnsibleParam -obj $params -name "enabled" -type "bool" -default $true
 $path = Get-AnsibleParam -obj $params -name "path" -type "str" -default '\'
 
+# Ensure the TaskPath starts with and ends with slashes as this causes issues with some
+# versions of Windows
+if ($path[0] -ne '\') {
```

Comment:
@dagwieers except that how would it ever be possible to get anything but the root level directory?  Relative addressing requires something to be relative to and there is no possible option to set in this case.